### DISCUSSION TITLE
Can I see my Python source code within the Dagster UI?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17959] [Q&A] [answered]
Hi team,

Can source code of an asset be displayed in Dagster UI?
In Airflow UI, users can view source code of a DAG. Not sure if Dagster has similar feature.

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699801382025299?thread_ts=1699801382.025299&cid=C01U954MEER).*

**Comment by alangenfeld** [comment, marked as answer]
Dagster does not currently have a similar feature, relevant issue is https://github.com/dagster-io/dagster/issues/8939
